selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java

```diff
@@ -158,4 +159,14 @@ private void createNewJavaCommand() {
     commandsExplorer.waitCommandInExplorerByName(JAVA_NAME);
     commandsEditor.waitTabIsPresent(JAVA_NAME);
   }
+
+  private void runCommandWithCheckResult() {
+    commandsEditor.clickOnRunButton();
+    try {
+      consoles.waitExpectedTextIntoConsole(PATH_TO_ROOT_FOLDER);
+    } catch (TimeoutException ex) {
+      commandsEditor.clickOnRunButton();
```

Comment:
It's better to add comment about which problem this workaround is aimed to solve. 
Even better variant - enclose verification if run button click affects properly inside the **clickOnRunButton**() method.